app/src/main/java/com/cmput301w23t40/capturetheqr/DB.java

```diff
@@ -120,6 +120,11 @@ static protected void saveQRCodeInDB(QRCode qrCode){
         // FIXME DB
     }
 
+    static protected void getQRCodeInDB(String hash, String location){
```

Comment:
I'm assuming the void return type is temporary so that it doesn't complain?

Reply:
yeah good catch, just don't know what object it would be so I assume this is ok for now